### DISCUSSION TITLE
Hide Moderation button of own posts

### DISCUSSION THREAD

**Discussion by yafkari** [forem/forem#15957] [Feature Requests]
**Describe the bug**

When a moderator is navigating to one of his own post, he can still click on the moderation button which does not make sense. We should not be able to moderate our own articles (imo)

**To Reproduce**

Create a post as a moderator and click on the moderation button.

**Expected behavior**

Should be hidden or at least showing a message saying like "You cannot moderate one of your own articles".

**Comment by citizen428** [comment]
I converted this to a discussion now but for the record, I'm very much in favor of this proposed changed 💯
